fpdf/table.py

```diff
@@ -83,6 +88,14 @@ def __init__(
         self._width = fpdf.epw if width is None else width
         self._wrapmode = wrapmode
         self.rows = []
+
+        from fpdf.fpdf import get_padding_tuple  # Avoid circular import
+
+        if padding is None:
+            self._padding = get_padding_tuple(0.5 * self._line_height)
```

Comment:
Why using this default value?

Unless you have a very good reason to do, I'd really prefer not to break this in term of backward compatibility,
as this change could cause all tables built by `fpdf2` users since v2.7.0 to overflow to another page!

Reply:
agreed. backwards compatibility is better than a default value